src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseHandler.cs

```diff
@@ -89,6 +98,33 @@ static DatabaseHandler()
             displayFileTypes.TryAdd(FileType.Log, SR.prototype_file_logFile);
             displayFileTypes.TryAdd(FileType.FileStream, SR.prototype_file_filestreamFile);
 
+            displayOperationModeOptions.TryAdd(QueryStoreOperationMode.Off, CommonConstants.QueryStoreOperationMode_Off);
```

Comment:
I don't think we should be localizing option values like OFF and READ_ONLY - do we do that in SSMS?

@erinstellato-ms what do you think?

Reply:
Other couple of dropdown options(All,Auto,None,Custom) are translated in the MS quey store help doc, so localized these too.

Reply:
What's our general practice on localizing options in ADS?  Specifically, options in a drop down?  Example: Backup functionality (in preview), we have Set backup compression as a drop down with three options.  Are those localized?

Fundamentally...first, let's be consistent across ADS.  Second, I will have to install another language of SSMS to see what we do there :)

Reply:
I have created a task to verify and add/remove all such dropdown options:
https://github.com/microsoft/sqltoolsservice/issues/2213